packages/mongo/collection.js

```diff
@@ -1000,12 +999,13 @@ Object.assign(Mongo.Collection.prototype, {
    * @memberof Mongo.Collection
    * @instance
    * @param {MongoSelector} selector Specifies which documents to remove
+   * @param {Function} [callback] Optional.  If present, called with an error object as the first argument and, if no error, the number of affected documents as the second.
    */
-  remove(selector) {
+  remove(selector, callback) {
     selector = Mongo.Collection._rewriteSelector(selector);
 
     if (this._isRemoteCollection()) {
-      return this._callMutatorMethod('remove', [selector]);
+      return this._callMutatorMethod('remove', [selector], callback);
```

Comment:
We have decribed in the documentation [the callback option for remove behavior](https://docs.meteor.com/api/collections#Mongo-Collection-remove). However, the code was not properly passing. Feels weird that this was not working and there is no test coverage, at least on Meteor 3.0. Now it does.